### PR TITLE
Determine which dota map to display from the patch the match was played in

### LINKED ISSUE #1299: vision tab still uses 7.06 map
the vision tab still uses the  7.06 mini map instead of the new 7.07 map.

### PR DESCRIPTION
**Pull request by jayantak**
fixes #1299

### CODE REVIEW

File: src/components/DotaMap/index.jsx [hunk 1]
```diff
@@ -9,15 +10,26 @@ const setMapSizeStyle = (width, maxWidth) => ({
...
-const isPost700 = unixDate => unixDate > 1481648400;
+const dotaMaps = [
+  { patchName: '7.07', mapImage: '/assets/images/dota2/map/detailed_707.png' },
+  { patchName: '7.00', mapImage: '/assets/images/dota2/map/detailed_700.png' },
+  { patchName: '6.86', mapImage: '/assets/images/dota2/map/detailed_686.png' },
+  { patchName: '6.82', mapImage: '/assets/images/dota2/map/detailed_682.png' },
+  { patchName: '6.70', mapImage: '/assets/images/dota2/map/detailed_pre682.png' },
+];
+
+const defaultMap = '/assets/images/dota2/map/detailed.png';
```
Comment (howardchung) [inline src/components/DotaMap/index.jsx]: I think this is the pre_700 map from before.  Now we have more accurate maps.

I think the default should be the latest map `dotaMaps[0].mapImage`, and we should delete detailed.png

Comment (jayantak) [reply src/components/DotaMap/index.jsx]: Okay, sounds good

Comment (jayantak) [reply src/components/DotaMap/index.jsx]: Done

File: src/components/DotaMap/index.jsx [hunk 2]
```diff
@@ -9,15 +10,25 @@ const setMapSizeStyle = (width, maxWidth) => ({
...
-// TODO make this a generic function that can determine which of multiple maps to use
 const getUrl = (startTime) => {
-  if (startTime === null || isPost700(startTime)) {
-    return '/assets/images/dota2/map/detailed_700.png';
+  if (startTime == null) return dotaMaps[0];
+  for (let i = 0; i < dotaMaps.length; i += 1) {
+    if (startTime >= patchDate[dotaMaps[i].patchName]) return dotaMaps[i].mapImage;
   }
-  return '/assets/images/dota2/map/detailed.png';
+  return dotaMaps[0];
```
Comment (howardchung) [inline src/components/DotaMap/index.jsx]: .mapImage

Comment (jayantak) [reply src/components/DotaMap/index.jsx]: Good catch, fixed

**Pull request merged**